src/averageAges.js

```diff
@@ -15,11 +15,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const cemetery = !century
+    ? people.filter(human =>
+      human.sex === 'm')
+
+    : people.filter(human =>
+      Math.ceil(human.died / 100) === century
+      && human.sex === 'm');
+
+  const callback = (allManAges, man) => {
+    const manAge = man.died - man.born;
+
+    return allManAges + manAge;
+  };
```

Comment:
There's no real need to create separate callback function. Write it inside reduce method.

```suggestion
  const callback = (allMenAges, man) => {
    const manAge = man.died - man.born;

    return allMenAges + manAge;
  };
```